.github/workflows/web-conformance.yaml

```diff
@@ -57,3 +57,22 @@ jobs:
             ${{ runner.os }}-connect-web-safari-conformance-
       - name: testwebsafariconformance
         run: make testwebsafariconformance
+  node:
+    runs-on: ubuntu-22.04
```

Comment:
We're running conformance tests for `@connectrpc/connect-web` on Node.js now. They run quick enough, seems worth it.